common/locales/en/defaultTasks.json

```diff
@@ -11,6 +11,20 @@
   "defaultTodoNotes": "You can either complete this To-Do, edit it, or remove it.",
 
   "defaultTodo1Text": "Join Habitica (Check me off!)",
+  "defaultTodo1Notes": "You can either complete this To-Do, edit it, or remove it.",
+
+  "defaultTodo2Text": "Learn About Habitica",
+  "defaultTodo2Notes": "Overview of New Users:\n\n💻 only: ☰(Menu - Top Bar) > Help\n\nFAQ:\n\n📱: ☰ > Help & FAQ\n\n💻 : ☰ > Help > FAQ",
+  "defaultTodo2Checklist1": "Read the Overview of New Users (Website only)",
+  "defaultTodo2Checklist2": "Read the FAQ",
+
+  "defaultTodo3Text": "Join the Newbies Guild",
+  "defaultTodo3Notes": "☰ > Social > Guilds/n/nThis is the best place to ask questions!",
```

Comment:
Your `/n` should be `\n`

![screen shot 2016-09-12 at 8 08 20 am](https://cloud.githubusercontent.com/assets/2916945/18436887/2978c518-78c0-11e6-8c8a-a9f033a709ff.png)